std/utf.d

```diff
@@ -4361,18 +4374,60 @@ if (isSomeChar!C)
                     buff = Empty;
                 }
 
-                static if (isForwardRange!R)
+                static if (isBidirectionalRange!R)
```

Comment:
you could move the `static if` inside the save function:

```d
static if (isForwardRange!R)
{
    auto save()
    {
         static if (isBidirectionalRange!R) { ... }
         else  { ... }
    }

Reply:
@RazvanN7 I changed it, I fixed the rebase and I added an unittest for the save() function. I'm sorry for the rebasing mistake.